src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -118,17 +118,45 @@
   </div>
   <div class="section-body">
     <table>
-      <tr>
-        <td></td>
-        <td>{{ ccj.claimantName}}</td>
-      </tr>
+      {% if ccj.signerName is empty %}
+        <tr>
+          <td>
+            Name:
+          </td>
+          <td>{{ ccj.claimantName}}</td>
+        </tr>
+      {% endif %}
+
+      {% if ccj.signerName is not empty %}
+        <tr>
+          <td>
+            Organisation:
+          </td>
+          <td>{{ ccj.claimantName }}</td>
```

Comment:
this doesn't look right

Reply:
WTF

Reply:
```
        {% if claimant.fullName is not empty %}
        <td>{{ claimant.fullName }}</td>
        {% endif %}
        {% if claimant.businessName is not empty %}
        <td>{{ claimant.businessName }}</td>
        {% endif %}
```

Reply:
I feel like you could have checked the copy paste here yourself :(

Reply:
https://github.com/hmcts/cmc-claim-store/blob/66f0c339a345731d717cefd6af3c6179c19ca755/src/main/resources/staff/templates/document/sealedClaim.html

Reply:
I don't usually do copy paste

Reply:
**Individual and SoleTraders:**

{% if ccj.signerName is empty %}
        <tr>
          <td>
            Name:
          </td>
          <td>{{ ccj.claimantName}}</td>
        </tr>
      {% endif %}

**Company or Organisation:**

      {% if ccj.signerName is not empty %}
        <tr>
          <td>
            Organisation:
          </td>
          <td>{{ ccj.claimantName }}</td>
        </tr>

        <tr>
          <td>
            Name:
          </td>
          <td>{{ ccj.signerName }}</td>
        </tr>

        <tr>
          <td>
            Position held:
          </td>
          <td>{{ ccj.signerRole }}</td>
        </tr>
      {% endif %}

Reply:
You had discussion with Ben in which I wan't involved, Ideally I expect you to update story or add a comment instead of going back and forth :(

Reply:
My discussion from ben's outcome:
Do exactly what claim and response do.
There has been no back and forward.

If you aren't clear on anything please say so

Reply:
That is not very helpful, do exactly as claim and response, what does it mean really. CCJ is totally different :(

Reply:
how is it totally different?
it should be exactly the same with different text for the declaration.

you have a party type
you have a claimantName you may have a business name
you have a signer name
you have a signer role

Reply:
changes I have done should produce PDF with following content and Ben confirm it

Declaration
[Claimant name]
[Organisation/Company Name] 
[Signer Name]
[Signer Role]
I declare that the details I have given are true to the best of my knowledge.
[date of request of CCJ]

Reply:
You are missing this check:
https://github.com/hmcts/cmc-claim-store/blob/66f0c339a345731d717cefd6af3c6179c19ca755/src/main/resources/staff/templates/document/sealedClaim.html#L360

Reply:
don't need that check, for company and organisation name is mandatory, isn't it?

Reply:
hmm must be a mistake on the sealed claim form then...